app/main.py

```diff
@@ -1,6 +1,17 @@
-from typing import Callable
+from typing import Callable, Any
+import functools
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    functools.wraps(func)
```

Comment:
Place a `wraps` decorator at the right place as [provided in the documentation](https://docs.python.org/uk/3/library/functools.html#functools.wraps).